content/Assets/Styles/print/_h5bp-print.scss

```diff
@@ -0,0 +1,78 @@
+/* ==========================================================================
```

Comment:
I'd suggest renaming filename to `_h5bp.scss`, it's implied it's "print" by the folder it's in.